core/pom.xml

```diff
@@ -270,7 +270,7 @@
     <dependency>
       <groupId>org.tachyonproject</groupId>
       <artifactId>tachyon-client</artifactId>
-      <version>0.8.1</version>
+      <version>0.8.2</version>
       <exclusions>
         <exclusion>
           <groupId>org.apache.hadoop</groupId>
```

Comment:
I've filed https://tachyon.atlassian.net/browse/TACHYON-1380 to ask Tachyon to change the handling of dependnecies in this POM for future releases. For now, though, I've verified that we don't need to change any of these exclusions because the client's dependencies haven't changed, so I think this PR should be good to merge.